src/App.js

```diff
@@ -1,12 +1,9 @@
-import * as React from 'react';
 import Calculator from './components/Calculator';
 
-export default class App extends React.PureComponent {
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
-      </div>
-    );
-  }
+export default function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
 }
```

Comment:
- [ ] It would be better if you use the arrow function instead of the function keyword as one of the general requirements for JavaScript best practices. Kindly fix this minor issue. 

😄 💪

Reply:
Hi, @BrianSammit  thank you for your recommendation, I tried to use arrow functions before but the problem is that it shows a linter error. 
![image](https://user-images.githubusercontent.com/28109626/138349861-d64d6a8b-cb37-4daf-b485-6d95a6a17d65.png)
 the linter error, in this case, is "Component definition is missing display name".

Now if I try to add a name like this 
![image](https://user-images.githubusercontent.com/28109626/138350308-83ad8838-8193-4560-8d78-5b3089212c3b.png)
the linter error, in this case, is "Parsing error: Only expressions, functions or classes are allowed as the `default` export. "

There is a way that I can implement arrow functions without getting linter errors? :thinking:  , thanks

Reply:
I forgot this case 
![image](https://user-images.githubusercontent.com/28109626/138350965-1b1597a5-cb78-4742-9fa0-5e1125dcaa7d.png)
"Prefer default export"